src/index.html

```diff
@@ -7,12 +7,76 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <a
+        class="card__link-img"
+        href="#product"
+      >
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="Product image"
```

Comment:
- alt attribute should describe the image content
<img width="500" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/63243136/5b11c898-1139-48dc-8bdd-df740ac61a90">

Reply:
Fixed